### PR TITLE
add wallets

### PR DESCRIPTION
**Pull request by mrval5**
### What does this do?
We added more wallets: WalletConnect, Coinbase, Fortmatic, Portis

### How do I test this?
Clicking on connect button

### CODE REVIEW

**Comment by AdamDotCom** [comment]
Nice! That was quick. 

Presently when we click on the MetaMask option is opens up MetaMask and prompts for credentials. Can we do something similar for the Walletconnect, Coinbase, Fortmatic, and Portis? Our Wilder World Market app has the functionality we're looking for (click the 'Connect' button at https://app.wilderworld.com/market/).

**Comment by AdamDotCom** [comment]
We're making use of [web3-react](https://github.com/NoahZinsmeister/web3-react) package. I suspect we'll need to add additional configuration values for the new providers in this file: https://github.com/zer0-os/zOS/blob/49db8361ea30991833d129623096c8ab0206cd8c/src/lib/web3/web3-react.tsx#L42-L42

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ on:
     branches-ignore:
       - main
       - development
+env:
+  REACT_APP_INFURA_URL: https://kovan.infura.io/v3/number123
```
Comment (mrval5) [inline .github/workflows/run-tests.yml:8]: this line is added to fix the unit test.

the lib is expecting an url:
https://github.com/NoahZinsmeister/web3-react/blob/v6/packages/network-connector/src/index.ts#L32

File: src/lib/web3/connectors.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  appName: 'zOS',
+});
+
+export const fortmatic = new FortmaticConnector({
+  chainId: Chains.Kovan,
+  apiKey: 'api_key',
+});
+
+export const portis = new PortisConnector({
+  dAppId: config.rootDomainId,
```
Comment (mrval5) [inline src/lib/web3/connectors.ts:32]: @AdamDotCom I'm not sure about the value of `dAppId `

**Comment by AdamDotCom** [comment]
This looks great. Were you able to manually verify that the new wallet connections? We don't have a dedicated QA team so the verification testing falls on us.

**Review by AdamDotCom — APPROVED** [review]

**Comment by mrval5** [comment]
> This looks great. Were you able to manually verify that the new wallet connections? We don't have a dedicated QA team so the verification testing falls on us.

I'm able to connect with wallets

**Review by AdamDotCom — APPROVED** [review]

**Pull request merged**